### DISCUSSION TITLE
Renaming playcover-community to playcover

### DISCUSSION THREAD

**Discussion by hayatofujii** [Homebrew/discussions#3642] [Casks]
Hello!

A while ago I submitted a community-based `playcover` cask. Although the "original" PlayCover [still floats around ](https://www.patreon.com/playcoverapp), it's kinda of paywalled, so I changed the cask name to `playcover-community` as suggested by maintainers.

The project now asks if a rename it back just to playcover if is feasible/adequate given the current situation. I don't think it's fair to pull request without discussing the situation beforehand.

**Comment by SMillerDev** [comment]
I disagree, a pull request is a much better way to discuss a change.
